pkg/cert/x509.go

```diff
@@ -111,3 +111,28 @@ func UnmarshalNutsDomain(data []byte) (string, error) {
 	}
 	return string(value.Bytes), nil
 }
+
+var ErrSANNotFound = errors.New("subject alternative name not found")
+
+// VendorIDFromCertificate returns the Nuts Vendor ID from a certificate.
+func VendorIDFromCertificate(certificate *x509.Certificate) (string, error) {
```

Comment:
I'd swear I already added this somewhere but couldn't find it, interesting. Anyways, good addition 👍 

Wondering though, as the vendor ID should be a PartyID (from nuts-go-core), should we return that already here? Then the caller is sure it gets a properly formatted vendor ID.

Reply:
There's an extraction function, but it's very low level.

Will return PartyID